cmd/goQuery/commands/root.go

```diff
@@ -82,17 +85,34 @@ func init() {
 
 	// Integers
 	rootCmd.Flags().IntVarP(&cmdLineParams.NumResults, "limit", "n", query.DefaultNumResults, helpMap["NumResults"])
-	rootCmd.Flags().IntVarP(&cmdLineParams.ResolveRows, "resolve-rows", "", query.DefaultResolveRows, helpMap["ResolveRows"])
-	rootCmd.Flags().IntVarP(&cmdLineParams.ResolveTimeout, "resolve-timeout", "", query.DefaultResolveTimeout, helpMap["ResolveTimeout"])
+	rootCmd.Flags().IntVarP(&cmdLineParams.DNSResolution.MaxRows, "resolve-rows", "", query.DefaultResolveRows, helpMap["ResolveRows"])
+	rootCmd.Flags().DurationVarP(&cmdLineParams.DNSResolution.Timeout, "resolve-timeout", "", query.DefaultResolveTimeout, helpMap["ResolveTimeout"])
 	rootCmd.Flags().IntVarP(&cmdLineParams.MaxMemPct, "max-mem", "", query.DefaultMaxMemPct, helpMap["MaxMemPct"])
 	rootCmd.Flags().BoolVarP(&cmdLineParams.LowMem, "low-mem", "", false, helpMap["LowMem"])
 
 	// Duration
 	rootCmd.Flags().DurationVarP(&cmdLineParams.QueryTimeout, "timeout", "", query.DefaultQueryTimeout, helpMap["QueryTimeout"])
 }
 
+func initLogger() {
```

Comment:
This should do the trick. Hard to test on a DB without errors, though. Help in verifying this is appreciated.

Reply:
How about just messing up the first few bytes of any GPFile on your test DB, that should do. Or deleting one of the GPFiles?

Reply:
Done. Now looks as follows:
```
./goQuery -d db-v4-broken/ -i any -f -180d -n 10000000 sip,dip  -e json | jq
2023-03-06T17:31:16.711+0100    error   goDB/DBWorkManager.go:215       metadata file `db-v4-broken/eth0/1669161600/.blockmeta` missing {"app_name": "goQuery", "app_version": "8e02f0a7"}
Query execution failed: internal error during query processing
```

Reply:
Sweet, LGTM!